package.json

```diff
@@ -83,16 +83,17 @@
   "config": {
     "forge": {
       "packagerConfig": {
-        "asar": true
+        "asar": true,
+        "name": "Leaphy Easybloqs",
+        "icon": "./easybloqs-icon.ico"
       },
       "makers": [
         {
           "name": "@electron-forge/maker-squirrel",
           "config": {
-            "name": "easybloqs",
+            "name": "LeaphyEasybloqs",
```

Comment:
Can't we default to name in package.json (and maybe rename that to include 'leaphy') ? Minimizing places where we configure names helps consistency.
https://js.electronforge.io/maker/squirrel/interfaces/makersquirrelconfig#name

Reply:
Good point! Removed the redundant name and added productName to the package.json.